src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapConfigurationProvider.java

```diff
@@ -78,4 +78,6 @@ public int compareTo(VanityPathConfig o2) {
      * If <code>null</code> is returned, all paths are allowed.
      */
     List<VanityPathConfig> getVanityPathConfig();
+
+    List<String> getAliasPath();
```

Comment:
I think getAllowedAliasPaths would be better.Also please document what this returns, if returns is nullable and what that means, etc.

Reply:
+1 to `getAllowedAliasPaths()` or `getAllowedAliasLocations()`